src/PhpSessionPersistence.php

```diff
@@ -89,6 +93,63 @@ public function __construct(bool $nonLocking = false, bool $deleteCookieOnEmptyS
         $this->cookieSameSite = ini_get('session.cookie_samesite');
     }
 
+    /**
+     * @param array $session
+     * @return self
+     */
+    public static function fromConfigArray(array $session = [])
```

Comment:
Naming nitpick - ideally, this should be something like `$config` or `$sessionConfig`

Reply:
Sorry, this method should also be marked as `final` and have the return type `self`

Reply:
Method is now final and argument is called `$sessionConfig`.